helm_deploy/hmpps-probation-estate-api/values.yaml

```diff
@@ -32,7 +32,11 @@ generic-service:
   namespace_secrets:
     hmpps-probation-estate-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
-
+    rds-instance-output:
```

Comment:
Not sure if it is important, but I normally prefix the RDS instance secret with something, e.g. `pe-`